src/app/rundown-view/components/minishelf/mini-shelf.component.html

```diff
@@ -0,0 +1,20 @@
+<div
+  class="c-mini-shelf"
+  title="{{ segment.name }}"
+  (click)="executeAction()"
```

Comment:
Since the associated action is executed, when clicking on the mini shelf, it would be nice with some visual feedback that it is possible. It would be nice if the cursor changed to show that one could interact with the mini shelf and perhaps make a slight style variation on hover.